src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -19,7 +19,15 @@ const RenderMatchUp = props => {
     <>
       <Header displayMenu={true} title="Scribble Stadium" />
       {/**Temporary button to redirect to point share component */}
-      <Button onClick={goToPointShare}>Go To Point Share</Button>
+      <div style={{ textAlign: 'center' }}>
+        <Button
+          style={{ marginTop: '1rem' }}
+          className="back-btn"
```

Comment:
Good idea giving these buttons a className. If more styling is needed in the future, we can take out the inline style and put it with the class styling.